update.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+git show "master:nsxiv.1" | groff -mandoc -Thtml |
+  sed 's|^</head>|<link rel="stylesheet" href="index.css"><link rel="icon" type="image/png" href="https://raw.githubusercontent.com/nsxiv/nsxiv/master/icon/16x16.png" sizes="16x16"></head>|' > index.html
```

Comment:
```suggestion

css='<link rel="stylesheet" href="index.css">'
icon_src='https://raw.githubusercontent.com/nsxiv/nsxiv/master/icon/16x16.png'
icon='<link rel="icon" type="image/png" href="'"$icon_src"'" sizes="16x16">'

git show "master:nsxiv.1" | groff -mandoc -Thtml |
  sed 's|^</head>|'"${css}${icon}"'</head>|' > index.html
```

Reply:
Untested, but I think it should work. @GRFreire

Reply:
That's way cleaner, thanks